app/models/discovery_fact_name.rb

```diff
@@ -0,0 +1,9 @@
+class DiscoveryFactName < FactName
+  def origin
+    'Discovery'
+  end
+
+  def icon_path
+    "icons16x16/stub/darkred-d"
```

Comment:
For recognition purposes I used a icon used for Darwin recognition. I'm not sure if it's good to stay at that or use/create another one.

Reply:
I think that's perfectly ok, we don't have a specific discovery icon

Reply:
Although, can we use the `gear` icon, just to keep it consistent with the welcome page?

Reply:
Although, can we use the gear icon, just to keep it consistent with the welcome page?

Reply:
Well, I would like to use a some icon with D in the center (like Puppet one). I'm not sure what kind of icon do you think but I really don't care what kind icon it will be. I only want to have a D in there because it signals that fact came from Discovery.

What do you think? @rabajaj0509 @ares

Reply:
I am okay with D or Foreman icon. Not sure about gears, that looks weird compared to other facts on a single page.